### PR TITLE
check file size in downloading

### LINKED ISSUE #42: Failed to download full file
If there is any error occurs during the downloading of one file, there will be a non-complete file stored and will not be updated in the next time.
In downloading, the file will not be created directly. Instead, it will get from the net and then append the file. If it si stopped, there will be an incomplete file.
When check updated, only the time is checked but whether the file is complete is not checked. So the incomplete file will not be detected
The user will not know that there is a incomplete file until the user try to open that file.

### PR DESCRIPTION
**Pull request by wznmickey**
Fix #42 by checking the size together.

### CODE REVIEW

**Comment by BoYanZh** [comment]
Use tmp file while downloading to fix #42.

**Comment by wznmickey** [comment]
> Use tmp file while downloading to fix #42.

Changed.

File: canvassyncer/__main__.py [hunk 1]
```diff
@@ -36,13 +36,20 @@ async def downloadOne(self, src, dst):
...
                     return self.failures.append(f"{src} => {dst}")
                 num_bytes_downloaded = res.num_bytes_downloaded
-                async with aiofiles.open(dst, "+wb") as f:
-                    async for chunk in res.aiter_bytes():
-                        await f.write(chunk)
-                        self.tqdm.update(
-                            res.num_bytes_downloaded - num_bytes_downloaded
-                        )
-                        num_bytes_downloaded = res.num_bytes_downloaded
+                dst_temp = dst+".temp"
```
Comment (BoYanZh) [inline canvassyncer/__main__.py]: You need to format your code with `black`.  https://github.com/psf/black

**Review by BoYanZh — CHANGES_REQUESTED** [review]

File: canvassyncer/__main__.py (hunk 1 above)
Comment (wznmickey) [reply canvassyncer/__main__.py]: Formated

**Pull request merged**